SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -221,11 +221,19 @@ void IPreInitilizeLowLevelSystem.SetCallback(Action<LowLevelSystem, SystemCallba
         #endregion
 
         #region init/close
+        [Obsolete("Not sure why this method is in here, the system should already be initialized when passed in through the constructor")]
         public void Initialize(int maxchannels, LowLevelInitFlags flags)
         {
             _system.init(maxchannels, (INITFLAGS) flags, IntPtr.Zero).Check();
         }
 
+        [Obsolete("This should only be called when we've initialized a low level system without studio (I think)")]
```

Comment:
The key is that when you initialize Studio (and call GetLowLevelSystem) then you shouldn't call close on the LowLevelSystem or you get an error when releasing the Studio. So I assumed that close/release on the LowLevel should only be done when you initialize the LowLevel without Studio. But it's based on assumptions, I didn't test that part.